pkg/actionsmetrics/event_reader.go

```diff
@@ -64,6 +64,9 @@ func (reader *EventReader) ProcessWorkflowJobEvent(ctx context.Context, event in
 	runsOn := strings.Join(e.WorkflowJob.Labels, `,`)
 	labels["runs_on"] = runsOn
 	labels["job_name"] = *e.WorkflowJob.Name
+	labels["organization"] = *e.Repo.Owner.Login
```

Comment:
@hamishforbes I realized #2218 added the organization label with another value of `*e.Org.Name`. If we call it "organization", that might be the correct value.

However, I do think it might be a good idea to be able to see `*e.Repo.Owner.Login` via labels when the workflow job event is coming from a runner associated with a repository owned by a user, not an organization.

That said, would you like to include `*e.Repo.Owner.Login` as another label, like "owner"?